themes/classic/_dev/css/components/featuredproducts.scss

```diff
@@ -3,14 +3,48 @@ $product-description-height: 70px;
 
 #products,
 .featured-products,
-.product-accessories,
-.product-miniature {
+.product-accessories {
   .products {
     display: flex;
     flex-wrap: wrap;
     justify-content: flex-start;
   }
 
+  .products-section-title {
+    margin: 2.5rem 0;
+    font-weight: 500;
+    text-align: center;
+  }
+
+  .product {
+    padding: 0;
+
+    @at-root .page-index &, .page-search & {
+      width: 25%;
+      min-width: 250px;
+    }
+  }
+
+  .product-miniature {
+    margin: 0 0.8125rem;
+  }
```

Comment:
I submitted another PR where I did refacto the whole listing so we use bootstrap columns, it won't be useful anymore in the future because we will rely on bootstrap columns spacing

Reply:
@NeOMakinG I think we should merge your changes first and then I can rebase my changes and adjust it to your refacto. Just ping me when it is done.

Reply:
You're right. Here is the PR : https://github.com/PrestaShop/PrestaShop/pull/25556 if you want to follow it.